### DISCUSSION TITLE
Images don't shown - Security Certificate and  404 error

### DISCUSSION THREAD

**Discussion by JanusHL** [community/community#22962] [Other Feature Feedback, Questions, & Ideas]
<p>Hi to all,</p>
<p>Although there are more 404 page errors I didn’t found the solution for my problem.</p>
<p>I run a repository for months,  and without any reason all the images  have stopped seeing any more in readme.md files, even if I try to access the files directly, I see nothing, or in the case I try to download the image at first I’ve received a Security Certificate HSTS error. After fixed the browser to by-pass this error, I’ve reveived always a 404 error page. I was looking in other github pages with images with the same result, so it seems a security problem with the domain that contains the images: <em><a href="http://raw.githubusercontent.com" rel="nofollow noopener">raw.githubusercontent.com</a></em>  that my browsers do not know how to solve.</p>
<p>I’m using Firefox and Opera both failed…</p>
<p>This is one of the images that return the error below.</p>
<p>Thank you in advance</p>
<pre><code>https://raw.githubusercontent.com/JanusHL/pvcontrol/master/dds238/dds238.jpg
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/9d1a1b2241cd269a3c77b0bea9fe0129a0880831.jpeg" data-download-href="/uploads/short-url/mpMTtbvjjihueEcll6BRfgOr5U5.jpg?dl=1" title="github_404_error.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/40386599/181074064-4f88b7bf-051f-4413-9a58-9496cbcf6048.jpeg" alt="github_404_error.JPG" data-base62-sha1="mpMTtbvjjihueEcll6BRfgOr5U5" width="690" height="355" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/9d1a1b2241cd269a3c77b0bea9fe0129a0880831_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github_404_error.JPG</span><span class="informations">973×502 30.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/janushl">@janushl</a>,</p>
<p>Thank you for being here and our apologies for the delayed response, this issue should be solved now. Are you still having errors?</p>

**Comment by JanusHL** [comment]
<p>Hi,</p>
<p>It seems solved… images are seen again</p>
<p>Thank you for solve the problem.</p>
<p>Best regards</p>
<p>JanusHL</p>
